### ISSUE TITLE
Braintree Failing with Unkown Error

### ISSUE THREAD

**Issue by KevDev99** [FlutterFlow/flutterflow-issues#2535]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Getting an Unknown Error when trying to perform a payment over Paypal or Credit Card with the Braintree Integration

### Expected Behavior

Receiving success message or accurate error message

### Steps to Reproduce

1. add credit form to a page
2. add button with a braintree payment action and choose credit card as type
3. fill out credit card information and then hit submit
4. Error: Unknown error occured

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4wlPHqw4t1rt9G1rqBd8JVmSk7Mjs3bb80letEcCs9f5CvOpYUec/6P1FuZNe7TFZHfWemp14E3NXIhd/2VvdfGzqZfKk6/7l2TzrgWlOleKmTF7WeYGhSQZlhGH610Z2nhiJABPBqLnBhwzqAP9iyaBXYQI+yC0QwSq/LZO4=

### Context

can't make any payment

### Visual documentation

![Screenshot 2024-03-15 at 16 16 26](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/3ca5ce91-11a1-47c5-ae14-bf53e27aab5f/3f5f7a5c-57e4-475a-9de3-4beed46428ea?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS8zY2E1Y2U5MS0xMWExLTQ3YzUtYWUxNC1iZjUzZTI3YWFiNWYvM2Y1ZjdhNWMtNTdlNC00NzVhLTlkZTMtNGJlZWQ0NjQyOGVhIiwiaWF0IjoxNzEwNzY1NDIxLCJleHAiOjMzMjgxMzI1NDIxfQ.JhZn8CUhiGR6Oy2fXybUsx9icYwmQ4SfpfDSm0E156g)

### Additional Info

Firebase Functions are deployed and the integration looks totally fine.

### Environment

```markdown
- FlutterFlow version: v4.1.33
- Platform: iOS
- Browser name and version: Chrome Version 122.0.6261.129
- Operating system and version affected: macOS 14.3.1 Darwin 23.3.0
```

**Comment by bledfort** [comment]
@KevDev99 Hi, I think I have the same issue as you. Did you manage to resolve it somehow ? I tried everything with no success :(

**Comment by KevDev99** [comment]
@bledfort unfortunately not , my last response from the flutterflow team that i need to create a prod version of my app and try then again. (looks like its currently a known issue that it's not working in dev...) will give it a try in prod and provide feedback

**Comment by DanLika** [comment]
Hi @wenkaifan0720 
I have the same issue. I tested "drop in (all options)," and I can't make a test purchase on my real Android device. When I tapped on "make a payment," the app crashed instantly, and the connection broke. When I enable credit cards and PayPal, then I'm available to make payment (Firebase, 403 error reject it..., this is for testing purpose, test credentials). 

![Image](https://github.com/user-attachments/assets/058476f7-04f5-4ca1-961f-21b4971ac2a2).
